webapi/src/test/scala/org/knora/webapi/e2e/v1/ResourcesV1R2RSpec.scala

```diff
@@ -1250,45 +1177,181 @@ class ResourcesV1R2RSpec extends R2RSpec {
                 }
 
                 assert(utf8str == "another simple text")
+            }
+        }
 
+        "create resources from an XML import" in {
```

Comment:
Could you provide a test using the anything ontology with all the possible value types occurring?